integr8ly/cloud-resource-operator
# Default CRO to MultiAZ deployments

## Overview

Jira: https://issues.redhat.com/browse/INTLY-5055

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make cluster/seed/managed/postgres`
- Run `make cluster/seed/managed/redis
- Run `make run`
- Once created, log into the AWS console, navigate to `RDS` -> `DB Instances` click on the provisioned DB and navigate to `configuration` 
- Ensure `MultiAZ` is enabled, and the secondary zone is different to the primary zone
- Now navigate to `Elasticache`, and navigate to `Redis`, expand the menu on the created cache to expose details on the cache.
- Ensure `MultiAZ` is enabled, with 2 nodes with two Availability Zones
